src/routes/error-routes.js

```diff
@@ -0,0 +1,29 @@
+import log from '../log';
+
+// Handle error 404
+export function notFoundHandler(req, res, next) {
+  const err = new Error('Page not found');
+  err.status = 404;
+  next(err);
+}
+
+// Handle all other errors
+export function errorHandler(err, req, res, next) {
+  log.error(req.url, err);
```

Comment:
First parameter of log functions should always be a hash. That allows data to be queried at a lated date as bunyan will emit it as JSON rather than just a string.

e.g. `log.error({ url: req.url, err }, 'An error occurred serving route');